services/core-api/app/api/mines/mine/models/mine.py

```diff
@@ -237,7 +237,7 @@ def find_by_mine_name(cls, term=None, major=None):
                            column('mine_location_description'), column('mine_name'), column('mine_no'),
                            column('deleted_ind'), column('major_mine_ind'))
 
-        mines_q = select([mine_table]).where(mine_table.c.deleted_ind == False)
+        mines_q = select([mine_table]).where(mine_table.c.deleted_ind == False).limit(50)
```

Comment:
I think the old code has the additional requirement to skip the check if minespace_user. We can add a check here, eg.

```
if not (is_minespace_user()):
            mines_q = mines_q.limit(100)
```

Reply:
Good catch!  Done!